privacyIDEA-api/target/idp-plugin-privacyIDEA-api-0.0.1.pom.asc

```diff
@@ -1,16 +1,16 @@
 -----BEGIN PGP SIGNATURE-----
 
-iQIzBAABCAAdFiEE9chR/w1w5heEDr5rB2a5Hkr+j4sFAmTLevsACgkQB2a5Hkr+

```

Comment:
The files in target have been added to gitignore, so they should be removed

Reply:
[21ed4f0](https://github.com/privacyidea/shibboleth-plugin/pull/7/commits/21ed4f0f925a81580caa53da545f32d1a9ced0ad)